Volume/Views/MainView/HomeList.swift

```diff
@@ -83,8 +83,8 @@ struct HomeList: View {
                                     BigReadArticleRow.Skeleton()
                                 }
                             }
-                        case .results(let results):
-                            LazyHStack(spacing: 24) {
+                        case .reloading(let results), .results(let results):
+                            HStack(spacing: 24) {
                                 ForEach(results.trendingArticles) { article in
                                     BigReadArticleRow(article: article)
                                 }
```

Comment:
Reason for switching to HStack?

Reply:
When testing, I was getting occasional jumping/glitching with the Lazy stacks, but not with the regular ones